dev-lisp/sbcl/sbcl-2.2.4.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="A high performance Common Lisp compiler"
+DESCRIPTION="Steel Bank Common Lisp (SBCL) is a high performance Common \
+Lisp compiler. It is open source / free software, with a permissive license. \
+In addition to the compiler and runtime system for ANSI Common Lisp, it \
+provides an interactive environment including a debugger, a statistical \
+profiler, a code coverage tool, and many other extensions."
+HOMEPAGE="https://sbcl.org"
+COPYRIGHT="2002 Gerd Moellmann"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/sbcl/sbcl/archive/refs/tags/sbcl-$portVersion.tar.gz"
+SOURCE_DIR="sbcl-sbcl-$portVersion"
+CHECKSUM_SHA256="f9f85f1a2ab4f12068621dce5a7b7b9c43a5875b06d736b88cb1373ca2caed1f"
+PATCHES="sbcl-2.2.4.patchset"
+
+ARCHITECTURES="x86_64"
+SECONDARY_ARCHITECTURES="x86"
```

Comment:
Maybe check on 32bit if we could enable it @augiedoggie ?

Reply:
> Maybe check on 32bit if we could enable it @augiedoggie ?

I already checked. It doesn't build.

Reply:
i had tried building for 32-bit at some point however after some digging into the source, quite a few more changes will need handled upstream before 32-bit support will work for haiku, i'm afraid.

Reply:
OK, I think we can leave 32bit out for a while until someone could fix this. Maybe add an issue for it so it can be looked into at some later point. For now focus on x86_64 :)